### PR TITLE
chore: move into mono repo

### PR DESCRIPTION
**Pull request by wolfenrain**
## Description

Move everything in a mono repo in preparation of tha future

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,17 +1 @@
...
-doc/api/
-
-# Temporary Files
-.tmp/
-
-# Files generated during tests
-.test_coverage.dart
-coverage/
\ No newline at end of file
+.idea
```
Comment (felangel) [inline .gitignore:1]: Why did we leave this line in?

File: packages/flame_behaviors/pubspec.yaml [hunk 2]
```diff
@@ -2,6 +2,9 @@ name: flame_behaviors
 description: Flame Behaviors applies separation of concerns to game logic in the form of Entities and Behaviors, built by Very Good Ventures.
 version: 0.1.1
 homepage: https://github.com/VeryGoodOpenSource/flame_behaviors
+repository: https://github.com/VeryGoodOpenSource/flame_behaviors
+issue_tracker: https://github.com/VeryGoodOpenSource/flame_behaviors/issues
+documentation: https://github.com/VeryGoodOpenSource/flame_behaviors/tree/main/docs
```
Comment (felangel) [inline packages/flame_behaviors/pubspec.yaml:7]: 💯

**Review by felangel — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (wolfenrain) [reply .gitignore:1]: Because that is what Dart Frog had 😂

File: packages/flame_behaviors/pubspec.yaml (hunk 2 above)
Comment (wolfenrain) [reply packages/flame_behaviors/pubspec.yaml]: I did uncomment the documentation one so that the pipeline can succeed, but will be fixed in the next PR

**Review by felangel — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (renancaraujo) [reply .gitignore:1]: 👀

**Pull request merged**